plugins/cros-ec/fu-cros-ec-usb-device.c

```diff
@@ -532,6 +758,17 @@ fu_cros_ec_usb_device_write_firmware (FuDevice *device,
 		return FALSE;
 	}
 
+	if (self->in_bootloader) {
+		fu_device_set_custom_flags (device, "rw-written");
```

Comment:
Is this really true?  Is it not possible for a device to be in bootloader mode already but not have written rw yet (in a failure flow most likely).